templates/server/src/main/resources/archetype-resources/core/pom.xml

```diff
@@ -167,11 +167,20 @@
     </dependency>
 #end
 
-    <!-- Flyway for DB Schema version management and migration -->
+    <!-- DB Schema version management and migration -->
+#if ($dbMigration == 'flyway')
     <dependency>
       <groupId>org.flywaydb</groupId>
       <artifactId>flyway-core</artifactId>
     </dependency>
+#end
+
+#if ($dbMigration == 'liquibase')
```

Comment:
use `#elseif` instead?

Reply:
Using #elseIf was resulting in error i.e "package liquibase doesn't exist" so this is the alternative way which I implemented.